catalog/video/catalog-mpegts-transformer/src/main/resources/videoMetacardType.json

```diff
@@ -0,0 +1,271 @@
+{
+  "metacardTypes": [
+    {
+      "type": "VideoMetacardType",
+      "attributes": {
+        "checksum":{
+          "required":false
+        },
+        "timestamp":{
+          "required":false
+        },
+        "mission-id":{
+          "required":false
+        },
+        "platform-tail-number":{
+          "reqiured":false
+        },
+        "platform-designation":{
+          "required":false
+        },
+        "image-source-sensor":{
+          "required":false
+        },
+        "image-coordinate-system":{
+          "required":false
+        },
+        "sensor-location":{
+          "required":false
+        },
+        "sensor-true-altitude":{
+          "required":false
+        },
+        "slant-range":{
+          "required":false
+        },
+        "target-width":{
+          "required":false
+        },
+        "frame-center-location":{
+          "required":false
+        },
+        "frame-center-elevation":{
+          "required":false
+        },
+        "offset-corner-location":{
+          "required":false
+        },
+        "target-location":{
+          "required":false
+        },
+        "target-location-elevation":{
+          "required":false
+        },
+        "ground-range":{
+          "required":false
+        },
+        "platform-call-sign":{
+          "required":false
+        },
+        "event-start-time":{
+          "required":false
+        },
+        "operational-mode":{
+          "required":false
+        },
+        "corner-location":{
+          "required":false
+        },
+        "security-classification":{
+          "required":false
+        },
+        "classifying-country-coding-method":{
+          "required":false
+        },
+        "classifying-country":{
+          "required":false
+        },
+        "object-country-coding-method":{
+          "required":false
+        },
+        "object-country-codes":{
+          "required":false
+        }
+      }
+    }
+  ],
+  "attributeTypes":{
+    "checksum":{
```

Comment:
There's already a metacard attribute named `checksum`, so I don't think this will work. Also, the checksums in the KLV are only used to check each KLV packet's integrity - in other words, I don't think they provide any useful information that we'd want on the metacard.